pyproject.toml

```diff
@@ -74,6 +74,7 @@ dependencies = [
     "kubernetes==29.0.0",
     "pytest-asyncio==0.23.6",
     "psycopg2-binary==2.9.9",
+    "pytest-reportportal==5.4.1",
```

Comment:
it is dev dependency, is not it?

Reply:
what do you mean by that? is this the wrong place for the testing dependencies?

Reply:
yes

Reply:
if you move it to dev deps, the tests jobs should start to pass

Reply:
Hi @tisnik sorry to bother but where are the dev dependencies defined?